### ISSUE TITLE
File Open/Save doesn't work

### ISSUE THREAD

**Issue by jfgo20** [automatak/dnp3-simulator#1]
Nothing happens when clicking Open or Save menu.

**Comment by jadamcrain** [comment]
Yes, unfortunately the current open / save button is just a placeholder. I'd really like to have the ability to load/save from a config file, but haven't found the time to add it.
